### PR TITLE
Add unnecessary line

### PR DESCRIPTION
**Pull request by minh711**
Please except!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,3 +41,5 @@ Then fulfill the information:
 
 But seem like it impossible to create a pull request directly from a branch, may
 be it should be folk to some other branch first.
+
+This is a unnecessary line!
```
Comment (minh711-2) [inline README.md]: What this? It is so unnecessary! Please remove asap!

**Review by minh711-2 — CHANGES_REQUESTED** [review]
Please fix as commented!

**Comment by minh711** [comment]
Fix how?

**Comment by minh711-2** [comment]
Just delete the line lol

**Review by minh711-2 — APPROVED** [review]
Now it's good!

**Pull request merged**